swagger.yaml

```diff
@@ -2101,8 +2150,8 @@ definitions:
     properties:
       kind:
         type: string
-        enum: ['Cluster']
-        description: Indicates the type of this object. Will be 'Cluster' if this is a complete object or 'ClusterLink' if it is just a link.
+        enum: ['Cluster', 'AddHostsCluster']
```

Comment:
@filanov @avishayt  i feel that there is going to be a long ping-pong regarding the names in this epic. i suggest that we continue with pushing this PR so that me and @danielerez can continue. there are still a lot of PR for this topic, and we can fixs naming issue pretty quickly in any one of them once we reach universal agreement